src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -88,12 +94,43 @@ export const NviCandidatePage = ({ nviListQuery }: NviCandidatePageProps) => {
           <ErrorBoundary>
             <PublicRegistrationContent registration={registrationQuery.data} />
 
-            {nextCandidateIdentifier && offsetNextCandidate && (
+            {previousCandidateIdentifier && currentCandidateIndex && (
+              <IconButton
+                component={Link}
+                to={{
+                  pathname: getNviCandidatePath(previousCandidateIdentifier),
+                  state: {
+                    offsetCandidate: currentCandidateIndex - 1,
+                  },
+                }}
+                data-testid={dataTestId.tasksPage.nvi.previousCandidateButton}
+                title={t('tasks.nvi.previous_candidate')}
+                size="small"
+                sx={{
+                  display: { xs: 'none', sm: 'flex' },
```

Comment:
Noen skjermstørrelser hvor denne ikke kommer så godt ut av det. Kanskje vi bare skal skjule for `sm` også som en enkel fiks?
```suggestion
                  display: { xs: 'none', md: 'flex' },
```
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/aa967b36-45cc-4e78-8627-af91e8b25909)